api/lib/infrastructure/utils/string-utils.js

```diff
@@ -21,8 +21,19 @@ function splitIntoWordsAndRemoveBackspaces(string) {
     .value();
 }
 
+/**
+ * Normalize and uppercase a string, remove non canonical characters and sort the remaining characters alphabetically
+ * @param str the string to sanitize
+ * @returns {string} the sanitized and sorted string
```

Comment:
Les descriptions des paramètres ne sont pas nécessaires ?

```suggestion
 * @param {string} str
 * @returns {string}
```

Reply:
Done